client/blocks/inline-help/inline-help-search-card.jsx

```diff
@@ -59,52 +46,6 @@ const InlineHelpSearchCard = ( {
 		return () => window.clearTimeout( timerId );
 	}, [ cardRef, location ] );
 
-	const onKeyDown = ( event ) => {
-		// ignore keyboard access when manipulating a text selection in input etc.
-		if ( event.getModifierState( 'Shift' ) ) {
-			return;
-		}
-		// take over control if and only if it's one of our keys
-		if ( includes( [ 'ArrowUp', 'ArrowDown', 'Enter' ], event.key ) ) {
-			event.preventDefault();
-		} else {
-			return;
-		}
-
-		switch ( event.key ) {
-			case 'ArrowUp':
-				setPreviousResult();
-				break;
-			case 'ArrowDown':
-				setNextResult();
-				break;
-			case 'Enter': {
```

Comment:
It seems that we have dismissed the tracking the event. 

@retrofox any input on this?

Reply:
No really. Need to confirm but I think we are tracking events for both events (onKeypress and onClick) in two different calls

Reply:
Yes, when the item is selected by clicking it's tracked [here](https://github.com/Automattic/wp-calypso/blob/master/client/blocks/inline-help/inline-help-search-results.jsx#L86-L88). And I think this is another good side about this PR, unifies the code.